src/odemis/driver/orsay.py

```diff
@@ -1316,8 +1315,8 @@ def _updateTemperature(self, parameter=None, attr_name="Actual"):
         """
         if parameter is None:
             parameter = self._temperaturePar
-        if float(self._temperaturePar.Actual) == float(self._temperaturePar.Target):
-            logging.debug("Target temperature reached.")
+        # if float(self._temperaturePar.Actual) == float(self._temperaturePar.Target):
+        #     logging.debug("Target temperature reached.")
```

Comment:
why is this commented?

Reply:
Too many logs. It's still handy to know how to check that the target temp is reached, so I left the code.